config/server.go

```diff
@@ -88,8 +112,8 @@ func _getHighestIP(ifs []net.Interface) (net.IP, error) {
 	return nil, fmt.Errorf("No non localhost IPv4 address found on interface lo")
 }
 
-func getLoopbackIP() (net.IP, error) {
-	iface, err := net.InterfaceByName("lo")
+func getLoopbackIP(g *Global) (net.IP, error) {
```

Comment:
Why don't you just pass the interface name (string)?

Reply:
Or define getLoopbackIP on `Global`